napari/_qt/menus/plugins_menu.py

```diff
@@ -30,21 +30,6 @@ def __init__(self, window: 'Window'):
         plugin_manager.events.unregistered.connect(
             self._remove_unregistered_widget
         )
```

Comment:
Not sure if I have done this part correctly.
What is the order in which the menu items are populated? i.e., is `self._add_menus()` in `napari/_qt/qt_main_window.py` run before or after the Q_PLUGINS_ACTIONS declared in `napari/_qt/_qapp_model/qactions/_plugins.py`? Does order matter or does the app model `Action`.
Also should these functions stay here or be moved to `qt_main_window.py` ?

Reply:
Chatted to @DragaDoncila and decided the discovery stuff and registering of plugin actions contained in `__init__.py` here should stay here (does not make sense to move to `qt_main_window.py`)